src/auth/dto/sign-in-user.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { IsString } from 'class-validator';
+
+export class SignInUserDto {
+  @IsString()
+  firstName: string;
+  @IsString()
+  password: string;
+}
```

Comment:
batter to couple the DTO and it's brother interface using "implements"
Using it we will understand each time we change the interface - we need to change DTO, because ther will be a TS error for us